packages/ui-components/src/components/info-label/info-label.tsx

```diff
@@ -0,0 +1,122 @@
+import { Component, Prop, h, Host, Element, State } from '@stencil/core';
+import { ResizeSensor } from 'css-element-queries';
+import clipboardHelper from '../../utils/clipboard.helper';
+import { EIconName } from '../icon/icon.types';
+import { COPY_TOOLTIP } from './info-label.types';
+
+@Component({
+	tag: 'kv-info-label',
+	styleUrls: {
+		night: 'info-label.night.scss',
+		light: 'info-label.light.scss'
+	},
+	shadow: true
+})
+export class KvInfoLabel {
+	private descriptionContainer: Element;
+	private tooltipConfig = COPY_TOOLTIP;
+	private resizeSensor: ResizeSensor = null;
+
+	/** Define if the show more button needed to be visible */
+	@State() enableShowMoreButton = false;
+	/** Define if the label content is completely visible */
+	@State() showMore = false;
+	/** Store the current div content height */
+	@State() currentDescriptionHeight: number = null;
+
+	/** (required) Info label title */
+	@Prop({ reflect: true }) labelTitle!: string;
+	/** (optional) Info label description */
+	@Prop({ reflect: true }) description?: string;
+	/** (optional) Info label description height */
+	@Prop({ reflect: true }) descriptionHeight?: number;
+	/** (optional) Info label description collapse text */
+	@Prop({ reflect: true }) descriptionCollapsedText = 'Read less';
```

Comment:
Move this string to `info-label.config.ts` as `DEFAULT_DESCRIPTION_COLLAPSED_TEXT`